english/expelliarmus/expelliarmus.md

```diff
@@ -0,0 +1,4 @@
+Use this disarming charm, also known as the Expelliarmus spell
+as a defensive tactic to force the marshmallow to release you.
+This has worked for Harry Pothead but may not for you.
+Given your current level of intoxication. In that case you may just need to take the [red pill](https://github.com/udacity/create-your-own-adventure/blob/master/english/red-pill/red-pill.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.